setup.py

```diff
@@ -1,12 +1,25 @@
-from pip.download import PipSession
+# PipSession is not available in older pips.
+try:
+    from pip.download import PipSession
+except:
+    # Precise has an old version of pip that lacks PipSession.
+    PipSession = None
 from pip.req import parse_requirements
 from setuptools import setup, find_packages
 
-pip_session = PipSession()
-requirements = parse_requirements("requirements.txt", session=pip_session)
-test_requirements = parse_requirements("test-requirements.txt",
-                                       session=pip_session)
+if PipSession is not None:
+    pip_session = PipSession()
+    kwargs = dict(session=pip_session)
+else:
+    # The parse_requirements in the precise version of pip is broken
+    # and requires options.skip_requirements_regex to be present.
+    from collections import namedtuple
+    options = namedtuple('Options', 'skip_requirements_regex')
+    options.skip_requirements_regex = None
```

Comment:
Hmm, it seems that you are setting a class attribute here. It works but probably it's not what you intended.
Perhaps:
```python
options = namedtuple('Options', 'skip_requirements_regex')
kwargs = dict(options=options(skip_requirements_regex=None))
```